editors/neovim/Portfile

```diff
@@ -35,17 +35,15 @@ depends_lib             port:gettext \
                         port:unibilium \
                         port:jemalloc \
```

Comment:
Neovim hasn't used jemalloc since at least v0.4.0. See https://github.com/neovim/neovim/pull/9526.

Reply:
Removed. Thank you!